documentation/development.md

```diff
@@ -0,0 +1,60 @@
+# Git Town Development
+
+## Requirements
+
+* Ruby 2.1.5
+  (install [directly](https://www.ruby-lang.org/en/documentation/installation),
+  or via a ruby manager like [rvm](https://rvm.io/)
+  or [rbenv](https://github.com/sstephenson/rbenv))
+* [ShellCheck](https://github.com/koalaman/shellcheck)
+
+
+## Setup
+
+* fork and clone the repository to your machine
+* `bundle` to install ruby gems
+
+
+## Running Tests
```

Comment:
Can we make this just say "Testing"? Simpler and fits with the one word headings better